pyaedt/edb_core/edb_data/padstacks_data.py

```diff
@@ -1394,6 +1394,39 @@ def name(self, value):
         self._edb_padstackinstance.SetName(value)
         self._edb_padstackinstance.SetProductProperty(self._pedb.edb_api.ProductId.Designer, 11, value)
 
+    @property
+    def metal_volume(self):
+        """Return the via hole instance metal volume in m3. Metal plating ratio is accounted.
```

Comment:
```suggestion
        """Metal volume of the via hole instance in cubic units (m3). Metal plating ratio is accounted.
```
Property description should be in as a "noun" not a "verb." What is meant by this: "Metal plating ratio is accounted."? can it be rephrased as "The metal plating ratio is accounted for in the metal volume."?